In [45]:
import numpy as np
import pandas as pd

# What You Are Answering
#Which of the 6 markets should Kayali prioritise for DTC or retail expansion?
#What is the opportunity score for each market based on multiple weighted factors?

Global Prestige Fragrance Market = $22.1 billion (Statista)
Total Global Fragrance Market = $60 billion (Grand View Research)

Prestige Share % = 22.1 ÷ 60 × 100 = 36.8% ≈ 37%

Step 1: Load Market_Data Sheet

In [46]:
market_df=pd.read_excel("Market Data.xlsx",sheet_name='Market_Data')

In [47]:
market_df.shape

(6, 15)

In [48]:
market_df.head()

,Market,Fragrance_Market_Size_USD_M,Growth_Rate_%,Prestige_Share_%,Search_Vol_Kayali_Perfume,Search_Vol_Arabian_Perfume,Search_Volume_Oud_Perfume,Kayali_Retailer_Present,Retailer Name,DTC_Site_Live,Source for search vol,Data_Year for recording the search vol,Date Year for market size,Source for Market Size,Notes
0,United States,"$13,041.7 M",0.047,0.37,110000,5400.0,8100,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
1,United Kingdom,"$3,356.8 M",0.044,0.37,49500,1300.0,12100,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
2,United Arab Emirates,"$2,020.4 M",0.077,0.37,8100,1900.0,3600,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
3,France,"$4,413.2 M",0.047,0.37,880,50.0,590,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
4,Germany,"$3,613.8 M",0.036,0.37,2400,NaN,1000,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...


In [49]:
market_df.dtypes

Market                                            object
Fragrance_Market_Size_USD_M                       object
Growth_Rate_%                                    float64
Prestige_Share_%                                 float64
Search_Vol_Kayali_Perfume                          int64
Search_Vol_Arabian_Perfume                       float64
Search_Volume_Oud_Perfume                          int64
Kayali_Retailer_Present                           object
Retailer Name                                     object
DTC_Site_Live                                     object
Source for search vol                             object
Data_Year for recording the search vol    datetime64[ns]
Date Year for market size                          int64
Source for Market Size                            object
Notes                                             object
dtype: object

In [50]:
market_df.columns.tolist()

['Market',
 'Fragrance_Market_Size_USD_M',
 'Growth_Rate_%',
 'Prestige_Share_%',
 'Search_Vol_Kayali_Perfume',
 'Search_Vol_Arabian_Perfume',
 'Search_Volume_Oud_Perfume',
 'Kayali_Retailer_Present',
 'Retailer Name',
 'DTC_Site_Live',
 'Source for search vol',
 'Data_Year for recording the search vol',
 'Date Year for market size',
 'Source for Market Size',
 'Notes']

Converting the market size to numeric

In [51]:
market_df['Fragrance_Market_Size_USD_M']=market_df['Fragrance_Market_Size_USD_M'].astype('str')

In [52]:
market_df['Fragrance_Market_Size_USD_M']=pd.to_numeric(market_df['Fragrance_Market_Size_USD_M'].str.replace('$','',regex=False).str.replace(',','',regex=False).str.replace('M','',regex=False).str.strip())

In [53]:
market_df['Fragrance_Market_Size_USD_M']

0    13041.7
1     3356.8
2     2020.4
3     4413.2
4     3613.8
5      540.0
Name: Fragrance_Market_Size_USD_M, dtype: float64

In [54]:
market_df['Growth_Rate_%']=market_df['Growth_Rate_%'].astype('str')

In [55]:
market_df['Growth_Rate_%']=pd.to_numeric(market_df['Growth_Rate_%'].str.replace('%','',regex=False).str.strip())

In [56]:
market_df['Growth_Rate_%']

0    0.0470
1    0.0440
2    0.0770
3    0.0470
4    0.0360
5    0.0046
Name: Growth_Rate_%, dtype: float64

In [57]:
market_df['Prestige_Share_%']=market_df['Prestige_Share_%'].astype('str')

In [58]:
market_df['Prestige_Share_%']=pd.to_numeric(market_df['Prestige_Share_%'].str.replace('%','',regex=False).str.strip())

In [59]:
market_df['Prestige_Share_%']

0    0.37
1    0.37
2    0.37
3    0.37
4    0.37
5    0.37
Name: Prestige_Share_%, dtype: float64

In [60]:
# FILLING GERMANY WITH 0 BECAUSE THE SEARCH VOL FOR ARABIAN PERFUME WILL BE 0 AND IT IS NOT POPULAR OVER THERE YET, SO FOR NORMALISING, NA IS REPLACED AS 0

In [61]:
market_df['Search_Vol_Arabian_Perfume']=market_df['Search_Vol_Arabian_Perfume'].fillna(0)

In [62]:
market_df

,Market,Fragrance_Market_Size_USD_M,Growth_Rate_%,Prestige_Share_%,Search_Vol_Kayali_Perfume,Search_Vol_Arabian_Perfume,Search_Volume_Oud_Perfume,Kayali_Retailer_Present,Retailer Name,DTC_Site_Live,Source for search vol,Data_Year for recording the search vol,Date Year for market size,Source for Market Size,Notes
0,United States,13041.7,0.0470,0.37,110000,5400.0,8100,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
1,United Kingdom,3356.8,0.0440,0.37,49500,1300.0,12100,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
2,United Arab Emirates,2020.4,0.0770,0.37,8100,1900.0,3600,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
3,France,4413.2,0.0470,0.37,880,50.0,590,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
4,Germany,3613.8,0.0360,0.37,2400,0.0,1000,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...
5,Netherlands,540.0,0.0046,0.37,1000,90.0,390,No,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,Statista Netherlands Fragrance Outlook 2024,Global prestige share 37% — calculated from St...


Normalisation

Normalisation puts every metric on the same 0–100 scale so each metric contributes to the score purely based on its weight — not its raw number magnitude.

In [63]:
market_df.head(1)

,Market,Fragrance_Market_Size_USD_M,Growth_Rate_%,Prestige_Share_%,Search_Vol_Kayali_Perfume,Search_Vol_Arabian_Perfume,Search_Volume_Oud_Perfume,Kayali_Retailer_Present,Retailer Name,DTC_Site_Live,Source for search vol,Data_Year for recording the search vol,Date Year for market size,Source for Market Size,Notes
0,United States,13041.7,0.047,0.37,110000,5400.0,8100,Yes,Sephora,Yes,Semrush Keyword Overview,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...


In [64]:
market_df['Norm_Fragrance_Market_Size']=((market_df['Fragrance_Market_Size_USD_M']-market_df['Fragrance_Market_Size_USD_M'].min())/(market_df['Fragrance_Market_Size_USD_M'].max()-market_df['Fragrance_Market_Size_USD_M'].min()))*100

In [66]:
market_df['Growth_Rate_%']=market_df['Growth_Rate_%']*100

In [67]:
market_df['Norm_Growth_Rate_%']=((market_df['Growth_Rate_%']-market_df['Growth_Rate_%'].min())/(market_df['Growth_Rate_%'].max()-market_df['Growth_Rate_%'].min()))*100

In [68]:
market_df['Norm_Search_Vol_Kayali_Perfume']=((market_df['Search_Vol_Kayali_Perfume']-market_df['Search_Vol_Kayali_Perfume'].min())/(market_df['Search_Vol_Kayali_Perfume'].max()-market_df['Search_Vol_Kayali_Perfume'].min()))*100

In [69]:
market_df['Norm_Search_Vol_Arabian_Perfume']=((market_df['Search_Vol_Arabian_Perfume']-market_df['Search_Vol_Arabian_Perfume'].min())/(market_df['Search_Vol_Arabian_Perfume'].max()-market_df['Search_Vol_Arabian_Perfume'].min()))*100

In [70]:
market_df['Norm_Search_Volume_Oud_Perfume']=((market_df['Search_Volume_Oud_Perfume']-market_df['Search_Volume_Oud_Perfume'].min())/(market_df['Search_Volume_Oud_Perfume'].max()-market_df['Search_Volume_Oud_Perfume'].min()))*100

In [ ]:
#Prestige share uniform across markets at 37% — normalised to neutral midpoint 50 as no market differentiation exists on this metric

In [72]:
market_df['Norm_Prestige_Share_%']=50

In [73]:
market_df['Prestige_Share_%']=0.37

In [74]:
market_df

,Market,Fragrance_Market_Size_USD_M,Growth_Rate_%,Prestige_Share_%,Search_Vol_Kayali_Perfume,Search_Vol_Arabian_Perfume,Search_Volume_Oud_Perfume,Kayali_Retailer_Present,Retailer Name,DTC_Site_Live,...,Data_Year for recording the search vol,Date Year for market size,Source for Market Size,Notes,Norm_Fragrance_Market_Size,Norm_Growth_Rate_%,Norm_Search_Vol_Kayali_Perfume,Norm_Search_Vol_Arabian_Perfume,Norm_Search_Volume_Oud_Perfume,Norm_Prestige_Share_%
0,United States,13041.7,4.70,0.37,110000,5400.0,8100,Yes,Sephora,Yes,...,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...,100.000000,58.563536,100.000000,100.000000,65.841161,50
1,United Kingdom,3356.8,4.40,0.37,49500,1300.0,12100,Yes,Sephora,Yes,...,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...,22.531336,54.419890,44.556452,24.074074,100.000000,50
2,United Arab Emirates,2020.4,7.70,0.37,8100,1900.0,3600,Yes,Sephora,Yes,...,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...,11.841590,100.000000,6.616569,35.185185,27.412468,50
3,France,4413.2,4.70,0.37,880,50.0,590,Yes,Sephora,Yes,...,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...,30.981387,58.563536,0.000000,0.925926,1.707942,50
4,Germany,3613.8,3.60,0.37,2400,0.0,1000,Yes,Sephora,Yes,...,2026-04-22,2024,grandviewresearch.com,Global prestige share 37% — calculated from St...,24.587056,43.370166,1.392962,0.000000,5.209223,50
5,Netherlands,540.0,0.46,0.37,1000,90.0,390,No,Sephora,Yes,...,2026-04-22,2024,Statista Netherlands Fragrance Outlook 2024,Global prestige share 37% — calculated from St...,0.000000,0.000000,0.109971,1.666667,0.000000,50


Section 3: Calculate Opportunity Score

Opportunity Score is a single number that summarises how attractive each market is for Kayali to expand into — combining multiple different factors into one comparable score.
Think of it like a report card for each market. Instead of looking at 6 different metrics separately and trying to mentally weigh them up — you combine them into one final grade.

In [75]:
weight_market_size = 0.25
weight_growth_rate = 0.20
weight_prestige_share = 0.15
weight_search_kayali = 0.20
weight_search_arabian = 0.10
weight_search_oud = 0.10

In [79]:
market_df.columns

Index(['Market', 'Fragrance_Market_Size_USD_M', 'Growth_Rate_%',
       'Prestige_Share_%', 'Search_Vol_Kayali_Perfume',
       'Search_Vol_Arabian_Perfume', 'Search_Volume_Oud_Perfume',
       'Kayali_Retailer_Present', 'Retailer Name', 'DTC_Site_Live',
       'Source for search vol', 'Data_Year for recording the search vol',
       'Date Year for market size', 'Source for Market Size', 'Notes',
       'Norm_Fragrance_Market_Size', 'Norm_Growth_Rate_%',
       'Norm_Search_Vol_Kayali_Perfume', 'Norm_Search_Vol_Arabian_Perfume',
       'Norm_Search_Volume_Oud_Perfume', 'Norm_Prestige_Share_%'],
      dtype='object')

In [81]:
market_df['Opportunity_Score']=(market_df['Norm_Fragrance_Market_Size'] * weight_market_size) +(market_df['Norm_Growth_Rate_%'] * weight_growth_rate)  +(market_df['Norm_Prestige_Share_%'] * weight_prestige_share) +(market_df['Norm_Search_Vol_Kayali_Perfume'] * weight_search_kayali) +(market_df['Norm_Search_Vol_Arabian_Perfume'] * weight_search_arabian) +(market_df['Norm_Search_Volume_Oud_Perfume'] * weight_search_oud)

In [83]:
market_df['Opportunity_Score'].max()

80.79682332237168

In [86]:
market_df['Opportunity_Score']=market_df['Opportunity_Score'].round(2)

In [87]:
market_df['Rank']=market_df['Opportunity_Score'].rank(ascending=False).astype(int)

In [89]:
market_df.sort_values(by='Opportunity_Score',ascending=False,inplace=True)

In [90]:
market_df

,Market,Fragrance_Market_Size_USD_M,Growth_Rate_%,Prestige_Share_%,Search_Vol_Kayali_Perfume,Search_Vol_Arabian_Perfume,Search_Volume_Oud_Perfume,Kayali_Retailer_Present,Retailer Name,DTC_Site_Live,...,Source for Market Size,Notes,Norm_Fragrance_Market_Size,Norm_Growth_Rate_%,Norm_Search_Vol_Kayali_Perfume,Norm_Search_Vol_Arabian_Perfume,Norm_Search_Volume_Oud_Perfume,Norm_Prestige_Share_%,Opportunity_Score,Rank
0,United States,13041.7,4.70,0.37,110000,5400.0,8100,Yes,Sephora,Yes,...,grandviewresearch.com,Global prestige share 37% — calculated from St...,100.000000,58.563536,100.000000,100.000000,65.841161,50,80.80,1
1,United Kingdom,3356.8,4.40,0.37,49500,1300.0,12100,Yes,Sephora,Yes,...,grandviewresearch.com,Global prestige share 37% — calculated from St...,22.531336,54.419890,44.556452,24.074074,100.000000,50,45.34,2
2,United Arab Emirates,2020.4,7.70,0.37,8100,1900.0,3600,Yes,Sephora,Yes,...,grandviewresearch.com,Global prestige share 37% — calculated from St...,11.841590,100.000000,6.616569,35.185185,27.412468,50,38.04,3
3,France,4413.2,4.70,0.37,880,50.0,590,Yes,Sephora,Yes,...,grandviewresearch.com,Global prestige share 37% — calculated from St...,30.981387,58.563536,0.000000,0.925926,1.707942,50,27.22,4
4,Germany,3613.8,3.60,0.37,2400,0.0,1000,Yes,Sephora,Yes,...,grandviewresearch.com,Global prestige share 37% — calculated from St...,24.587056,43.370166,1.392962,0.000000,5.209223,50,23.12,5
5,Netherlands,540.0,0.46,0.37,1000,90.0,390,No,Sephora,Yes,...,Statista Netherlands Fragrance Outlook 2024,Global prestige share 37% — calculated from St...,0.000000,0.000000,0.109971,1.666667,0.000000,50,7.69,6


In [91]:
status_map = {
    'United States': 'Established',
    'United Arab Emirates': 'Established',
    'United Kingdom': 'Active and Growing',
    'France': 'Expansion Target',
    'Germany': 'Expansion Target',
    'Netherlands': 'Expansion Target'
}

market_df['Market_Status'] = market_df['Market'].map(status_map)

In [92]:
print(market_df[['Market', 'Market_Status']])

                 Market       Market_Status
0         United States         Established
1        United Kingdom  Active and Growing
2  United Arab Emirates         Established
3                France    Expansion Target
4               Germany    Expansion Target
5           Netherlands    Expansion Target


In [93]:
conditions=[ market_df['Market_Status']== 'Established', market_df['Market_Status']== 'Active and Growing', (market_df['Market_Status']== 'Expansion Target') & (market_df['Opportunity_Score'] > 50), (market_df['Market_Status'] == 'Expansion Target') & (market_df['Opportunity_Score'] <= 50)]

In [94]:
choices = [
    'Optimise Existing',
    'Accelerate Growth',
    'High Priority Expansion',
    'Monitor — Lower Priority'
]

In [95]:
market_df['Expansion_Priority']=np.select(conditions,choices,default='Unknown')

In [96]:
market_df

,Market,Fragrance_Market_Size_USD_M,Growth_Rate_%,Prestige_Share_%,Search_Vol_Kayali_Perfume,Search_Vol_Arabian_Perfume,Search_Volume_Oud_Perfume,Kayali_Retailer_Present,Retailer Name,DTC_Site_Live,...,Norm_Fragrance_Market_Size,Norm_Growth_Rate_%,Norm_Search_Vol_Kayali_Perfume,Norm_Search_Vol_Arabian_Perfume,Norm_Search_Volume_Oud_Perfume,Norm_Prestige_Share_%,Opportunity_Score,Rank,Market_Status,Expansion_Priority
0,United States,13041.7,4.70,0.37,110000,5400.0,8100,Yes,Sephora,Yes,...,100.000000,58.563536,100.000000,100.000000,65.841161,50,80.80,1,Established,Optimise Existing
1,United Kingdom,3356.8,4.40,0.37,49500,1300.0,12100,Yes,Sephora,Yes,...,22.531336,54.419890,44.556452,24.074074,100.000000,50,45.34,2,Active and Growing,Accelerate Growth
2,United Arab Emirates,2020.4,7.70,0.37,8100,1900.0,3600,Yes,Sephora,Yes,...,11.841590,100.000000,6.616569,35.185185,27.412468,50,38.04,3,Established,Optimise Existing
3,France,4413.2,4.70,0.37,880,50.0,590,Yes,Sephora,Yes,...,30.981387,58.563536,0.000000,0.925926,1.707942,50,27.22,4,Expansion Target,Monitor — Lower Priority
4,Germany,3613.8,3.60,0.37,2400,0.0,1000,Yes,Sephora,Yes,...,24.587056,43.370166,1.392962,0.000000,5.209223,50,23.12,5,Expansion Target,Monitor — Lower Priority
5,Netherlands,540.0,0.46,0.37,1000,90.0,390,No,Sephora,Yes,...,0.000000,0.000000,0.109971,1.666667,0.000000,50,7.69,6,Expansion Target,Monitor — Lower Priority


In [99]:
market_df[['Market', 'Opportunity_Score', 'Rank', 'Market_Status', 'Expansion_Priority']].sort_values('Opportunity_Score', ascending=False)

,Market,Opportunity_Score,Rank,Market_Status,Expansion_Priority
0,United States,80.80,1,Established,Optimise Existing
1,United Kingdom,45.34,2,Active and Growing,Accelerate Growth
2,United Arab Emirates,38.04,3,Established,Optimise Existing
3,France,27.22,4,Expansion Target,Monitor — Lower Priority
4,Germany,23.12,5,Expansion Target,Monitor — Lower Priority
5,Netherlands,7.69,6,Expansion Target,Monitor — Lower Priority


In [100]:
market_df.to_excel('Market_Expansion_Detail.xlsx')

<ipython-input-100-df8475ef7a9b>:1: UserWarning: Pandas requires version '1.4.3' or newer of 'xlsxwriter' (version '1.2.9' currently installed).
  market_df.to_excel('Market_Expansion_Detail.xlsx')


Why Min-Max Is Right for Your Analysis

Reason 1: You Need a Bounded 0–100 Scale
Your Opportunity Score formula multiplies each normalised metric by a weight and adds them together. For this to produce a meaningful final score — every input metric must be on the same bounded scale.
Min-Max guarantees every value sits between 0 and 100. No exceptions.
Z-Score gives you values like -1.2, 0.4, 2.1 — unbounded and can be negative. Multiplying weights by negative numbers produces a nonsensical opportunity score.

Reason 2: The Output Is Intuitive and Explainable
Min-Max scores are immediately understandable to a non-technical person like the Global President:

"The US scores 100 on market size because it is the largest market. Netherlands scores 0 because it is the smallest."

That is a sentence anyone understands.
Z-Score scores are not intuitive:

"The US scores 2.3 standard deviations above the mean on market size."

A Global President does not know what that means and you would spend your presentation explaining the methodology instead of the insight.

Reason 3: You Have Only 6 Data Points
Z-Score standardisation works best with large datasets — ideally 30+ data points — because it relies on the mean and standard deviation being stable and representative.
With only 6 markets your mean and standard deviation are heavily influenced by individual outliers. The US market size of $8,800M would severely distort the mean and make Z-Score results unreliable.
Min-Max works perfectly with small datasets because it only needs the minimum and maximum values — both of which are clearly defined even with 6 data points.

Reason 4: You Want to Preserve Relative Differences
Min-Max preserves the proportional relationships between values.
Example:

US market size = 8800
France market size = 445
Netherlands market size = 245

After Min-Max:

US = 100 (largest)
France = 23.6
Netherlands = 0 (smallest)

The gap between US and France is preserved proportionally. You can genuinely say "the US market is dramatically larger than France" and the score reflects that.

Reason 5: Direct Comparability Across Metrics
After Min-Max normalisation a score of 80 on Market Size means exactly the same thing as a score of 80 on Search Volume — both mean "this market is 80% of the way between the worst and best performing market on this metric."
This direct comparability is essential for your weighted scoring formula to work correctly.